app/src/main/java/com/oracle/javafx/scenebuilder/app/SceneBuilderApp.java

```diff
@@ -124,7 +119,12 @@ public enum ApplicationControlAction {
     private final BooleanBinding startupTasksFinished = Bindings.isEmpty(startupTasks);
 
     static {
-        System.setProperty("java.util.logging.config.file", SceneBuilderApp.class.getResource("/logging.properties").getPath());
+        try {
+            // Reads logging property file from local runs, or embedded in a JAR
+            LogManager.getLogManager().readConfiguration(SceneBuilderApp.class.getResourceAsStream("/logging.properties"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Can we add a message in the runtime exception?

> Unable to load configuration file logging.properties

Reply:
Done there now with a second commit.